packages/common/src/decorators/__test__/attribute.decorator.spec.ts

```diff
@@ -60,3 +60,27 @@ test('adds raw metadata for enum property', () => {
     },
   ]);
 });
+
+/**
+ * Issue #29
```

Comment:
Is this a copy/paste typo?

Please remove, if it should be referring to this issue.